packages/camera/example/android/app/src/main/AndroidManifest.xml

```diff
@@ -1,30 +1,39 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
-    package="io.flutter.plugins.cameraexample">
+  package="io.flutter.plugins.cameraexample">
 
-    <uses-permission android:name="android.permission.INTERNET"/>
+  <application
+    android:icon="@mipmap/ic_launcher"
+    android:label="camera_example"
+    android:name="io.flutter.app.FlutterApplication">
+    <activity
+      android:configChanges="orientation|keyboardHidden|keyboard|screenSize|locale|layoutDirection"
+      android:hardwareAccelerated="true"
+      android:launchMode="singleTop"
+      android:name=".EmbeddingV1Activity"
+      android:theme="@style/LaunchTheme"
+      android:windowSoftInputMode="adjustResize">
+      <meta-data
+        android:name="io.flutter.app.android.SplashScreenUntilFirstFrame"
+        android:value="true"/>
+    </activity>
+    <activity
+      android:configChanges="orientation|keyboardHidden|keyboard|screenSize|locale|layoutDirection"
+      android:exported="true"
```

Comment:
i think it is supposed to be in the `EmbeddingV1Activity`

Reply:
Done.